### PR TITLE
PyData Markdown file

### PR DESCRIPTION
**Pull request by timvg80**
Here is a draft of the markdown file containing the relevant information for the PyData track

### CODE REVIEW

File: pages/pydata.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  * How existing Python packages helped you to achieve your project goals:
+    * Data preparation
+    * Statistical analysis
+    * Machine learning
+    * Data visualisation
+  * Have you been working on developing your own Python applications for data science?
+    * Even if these are a _work-in-progress_, this is a great forum to present these!
+    
+The time allocated for each talk is 30 minutes. If you are happy to receive any questions from the audience (this is not compulsory), then please allow time for these.
```
Comment (alexwlchan) [inline pages/pydata.md]: I could have sworn it was 25 minutes, but I could be wrong. I need to double check.

File: pages/pydata.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    
+## Previous PyData talks
+If anyone would like to see examples of the sorts of talks that have been presented at PyData conferences in UK the in the past, these can be seen on the [PyData YouTube channel](https://www.youtube.com/user/PyDataTV)
```
Comment (alexwlchan) [inline pages/pydata.md]: nit: full stop?

**Review by alexwlchan — APPROVED** [review]
Two very minor things, but otherwise LGTM.

Thanks @timvg80!

File: pages/pydata.md (hunk 1 above)
Comment (inglesp) [reply pages/pydata.md]: Yes: we've decided that talks are 25 minutes long.  Sorry for not telling you @timvg80.  We should add this to the  main CFP page too.

File: pages/pydata.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+title: PyData Track
+---
+
+This year, for the first time, the teams at PyCon and [PyData London](http://london.pydata.org/) are joining forces to deliver a data-science focussed track at the conference.
+***
+
+## What we are looking for
```
Comment (inglesp) [inline pages/pydata.md]: It's not clear that this section is about the CFP.  You could add a line to the paragraph above saying something like: "We will be hosting a number of talks and workshops about data science at the the conference, and we are looking for members of the PyData community to submit proposals to the [conference CFP](https://hq.pyconuk.org/cfp/proposals/new/)."

File: pages/pydata.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+This track is mainly focussed on looking for speakers who wish to share their experiences of using Python in data-science. Examples such as:
```
Comment (inglesp) [inline pages/pydata.md]: This feels a bit wordy -- could you change the opening to "We are looking for speakers..."?

**Review by inglesp — CHANGES_REQUESTED** [review]
Hi @timvg80, thanks for this.

I've made some suggested changes inline.

A couple of minor things: please can you always refer to the conference as "PyCon UK" to distinguish us from other PyCons; and can you decide on whether it's "data science" or "data-science"?

**Comment by timvg80** [comment]
Hi Alex and Peter. I have made some changes now, and have pushed them to my master branch.

File: pages/pydata.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+title: PyData Track
+---
+
+This year, for the first time, the teams at PyConUK and [PyData London](http://london.pydata.org/) are joining forces to deliver a data science focussed track at the conference. We will be hosting a number of talks and workshops about data science at the the conference, and we are looking for members of the PyData community to submit proposals to the [conference CFP](https://hq.pyconuk.org/cfp/proposals/new/).
```
Comment (alexwlchan) [inline pages/pydata.md]: nit: the conference name is “PyCon UK”, not “PyConUK”

(@inglesp: I’m sure we used to have CI checks for this?)

File: pages/pydata.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  * Have you been working on developing your own Python applications for data science?
+    * Even if these are a _work-in-progress_, this is a great forum to present these!
+    
+The time allocated for each talk is 25 minutes. If you are happy to receive any questions from the audience (this is not compulsory), then please allow time for these.
+    
+## Previous PyData talks
+If anyone would like to see examples of the sorts of talks that have been presented at PyData conferences in UK the in the past, these can be seen on the [PyData YouTube channel](https://www.youtube.com/user/PyDataTV).
+    
+## Community Spirit
+Both PyConUK and PyData are thriving communities that want to help people to learn and develop their tech skills. We welcome talks from all backgrounds and all abilities, across both academia and industry. Even if this your first talk at a tech conference, you will be given a warm welcome, with many people happy to provide any assistance or guidance that you require.
```
Comment (alexwlchan) [inline pages/pydata.md]: Up to you, but I wonder if it might be nice to have a link to http://2017.pyconuk.org/diversity-accessibility-inclusion/ in this paragraph?

**Review by alexwlchan — CHANGES_REQUESTED** [review]
Thanks, looks pretty good! A few more minor things.

File: pages/pydata.md (hunk 5 above)
Comment (timvg80) [reply pages/pydata.md]: Hi Alex... of course! I've fixed and pushed those changes now.

**Review by alexwlchan — APPROVED** [review]
Okay, I don’t think I have any other comments. Anything else from you, @inglesp?

**Review by inglesp — APPROVED** [review]

**Comment by timvg80** [comment]
Hi both,

Thanks for letting me know. I've just had a look now, and it looks great! I
have let the PyData team know in London as well. Just one thing... I
noticed that I had made an error in some word ordering in one of the
sections, so I have just corrected that and made another pull request.
Sorry for that.

Tim
